enamel/api/version.py

```diff
@@ -0,0 +1,86 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Microversion handling."""
+
+import collections
+
+# TODO(cdent): Get a real one, get it from config or other source of
+# defaults.
+SERVICE_TYPE = 'enamel'
+
+# The Canonical Version List
+VERSIONS = [
+    '0.1',
+    '0.9',
```

Comment:
Yeah, I meant to monkey that for the tests and forgot. Will do that when I add the route decorators.